src/training/stylegan.py

```diff
@@ -41,39 +41,44 @@ def __init__(self,
         self.total_epochs = total_epochs
         self.logger = logger
         self.storage = storage
-        self.ocr_loss = ocr_loss.to(device)
-        self.perceptual_loss = perceptual_loss.to(device)
-        self.typeface_loss = typeface_loss.to(device)
         self.ocr_coef = ocr_coef
+        self.cycle_coef = cycle_coef
+        self.recon_coef = recon_coef
         self.emb_coef = emb_coef
         self.perc_coef = perc_coef
         self.tex_coef = tex_coef
-        self.coef_ocr = ocr_coef
-        self.coef_cycle = coef_cycle_loss
-        self.coef_recon = coef_reconstruction_loss
+        self.typeface_loss = typeface_loss.to(device)
+        self.ocr_loss = ocr_loss.to(device)
+        self.perc_loss = perc_loss.to(device)
         self.style_embedder = style_embedder
         self.content_embedder = content_embedder
-        self.l1 = torch.nn.L1Loss() 
+        self.l1 = torch.nn.L1Loss()
```

Comment:
Don't we need to put in onto device? Also I am not sure that we should initialise it here and not in config.